tox.ini

```
@@ -14,8 +14,9 @@ deps =
     dj111: Django>=1.11,<2.0
     dj20: Django>=2.0,<2.1
     dj21: Django>=2.1,<2.2
+    dj22: Django>=2.2,<3.0
 
-    cms40: https://github.com/divio/django-cms/archive/release/4.0.x.zip
+    cms40: https://github.com/jonathan-s/django-cms/archive/django-2.2.zip
```

Needs to be replaced with the following once the patch for 2.2 is in place for django-cms. 

```suggestion
    cms40: https://github.com/divio/django-cms/archive/release/4.0.x.zip
```